src/test/java/tests/WebScraperTest.java

```diff
@@ -227,4 +227,21 @@ public void shouldReturn12MediaObjectsInSet() {
         assertReflectionEquals(expected, actual,
                 ReflectionComparatorMode.LENIENT_ORDER);
     }
+
+    /**
+     * This methods test that the depth returned from the getAllMediaObjects
+     * is 17.
+     * For Depth First search and the layout of the website,
+     * we determined that the depth of scraping all media objects will always be 17
+     */
+    @Test
+    public void crawlDepthGetAllMediaObjectsshouldReturn17() {
```

Comment:
name should be crawlDepthGetAllMediaObjectsShouldReturn17:

```suggestion
    public void crawlDepthGetAllMediaObjectsShouldReturn17() {
```